src/main/java/namor/template/calculator/ClassicCalculator.java

```diff
@@ -0,0 +1,87 @@
+package namor.template.calculator;
+
+import namor.template.stack.Stack;
+
+public class ClassicCalculator {
```

Comment:
You were coding this task by copying previous implementation, try to forget RPN and do here without assosiating your mind with previous task.